setup.cfg

```diff
@@ -0,0 +1,11 @@
+[flake8]
+ignore = E302 E305
+max-line-length = 90
+exclude = Django, env, results
+
+[isort]
+combine_as_imports = true
+include_trailing_comma = true
+line_length = 79
+multi_line_output = 5
+skip = Django, env, results
```

Comment:
Can you configure your editor to add the missing new lines at EOF? (Trim any trailing whitespace too is worth it.) 

(Why? Minimises diffs over the long-run.)